R/input-switch.R

```diff
@@ -73,6 +73,17 @@ toggle_switch <- function(id, value = NULL, session = get_current_session()) {
   session$onFlush(callback, once = TRUE)
 }
 
+# Keep this internal for now until we agree on the UX
+input_dark_mode_switch <- function(id, label, ..., width = "auto") {
+  res <- input_switch(id, label, ..., width = width)
+  # TODO: .navbar should probably do this for .shiny-input-container
```

Comment:
or would we add `.navbar` to the set of gap-spaced elements?

Reply:
I guess that's an option but also a bit weird considering it's not `flex-direction: column`